app/controllers/api/v2/bulk_searches_controller.rb

```diff
@@ -5,7 +5,7 @@ def create
         @result = ::BulkSearch::ResultCollection.enqueue(bulk_search_params)
         @serialized_result = Api::V2::BulkSearch::ResultCollectionSerializer.new(@result).serializable_hash
 
-        render json: @serialized_result, status: @result.http_code
+        render json: @serialized_result, status: :accepted
```

Comment:
Can we include a location with this - the logic being a lot of http clients will be able to follow a location header out of the box (eg Faraday can). This is standard practice for REST

Without it clients will require custom parsing logic to extract the url from the JSON body where they should be polling for the result. This isn't a lot of code to write but if we can follow standard practice, I think we should?